.appveyor.yml

```diff
@@ -1,3 +1,11 @@
+skip_commits:
+  files:
+    - ".github/workflows/docs.yml"
```

Comment:
```suggestion
```
Wouldn't ".github/workflows/docs.yml" by covered by ".github/workflows/*"?

Reply:
Yep, I actually forgot to push a change to `".github/**"`, now done!